src/api.ts

```diff
@@ -79,11 +83,15 @@ class Radar {
       Logger.debug('using options', options);
     }
 
-    // NOTE(jasonl): this allows us to run jest tests
-    // without having to mock the ConfigAPI.getConfig call
-    if (!(window as any)?.RADAR_TEST_ENV) {
-      ConfigAPI.getConfig();
+    if (!Config.isInitialized()) { // only call getConfig on first initialization
```

Comment:
Maybe we want to `|| keyChanged` to handle cases where someone would change the key (can't come up with anything off the top of my head where that would matter here, but we do allow the key to change in Waypoint for instance)

Reply:
Yeah, that's the one scenario I was sort of thinking of. 

Maybe it's worth just checking if the config options have changed or not.